Firestore/Source/Local/FSTLocalDocumentsView.mm

```diff
@@ -159,6 +171,26 @@ - (DocumentMap)documentsMatchingDocumentQuery:(const ResourcePath &)docPath {
   return result;
 }
 
+- (DocumentMap)documentsMatchingCollectionGroupQuery:(FSTQuery *)query {
+  HARD_ASSERT(query.path.empty(),
+              "Currently we only support collection group queries at the root.");
```

Comment:
Optional: consider adding a newline after this line, to separate precondition check.